docs-utils/update-images.js

```diff
@@ -22,26 +23,32 @@ const getExtension = (url) => {
 }
 
 const updateImages = async (filepath) => {
+  console.log("Running update images")
+  console.log(filepath)
```

Comment:
I think we could get rid of this log here. It has fulfilled its debugging purpose already.